crates/subspace-networking/src/connected_peers.rs

```diff
@@ -79,8 +79,9 @@ enum ConnectionState {
     NotInterested,
 }
 
-impl ToString for ConnectionState {
-    fn to_string(&self) -> String {
+impl ConnectionState {
+    /// Converts [`ConnectionState`] to a string with information loss.
+    fn stringify(&self) -> String {
```

Comment:
I literally read it the same way as `to_string`, but I am a victim of JavaScript: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

This is why I was trying to find a method name that doesn't imply encoding/serialization/etc.